### PR TITLE
[doc] Update README.md to include Graal repository

### PR DESCRIPTION
**Pull request by valeriocos**
This code modifies the README.md file to include the link to the GitHub repository hosting Graal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ Currently, GrimoireLab is organized in the following repositories:
 * [Perceval (bundle for OPNFV)](https://github.com/chaoss/grimoirelab-perceval-opnfv)
 * [Perceval (bundle for Mozilla)](https://github.com/chaoss/grimoirelab-perceval-mozilla)
 * [Perceval (bundle for Puppet)](https://github.com/chaoss/grimoirelab-perceval-puppet)
+* [Graal](https://github.com/chaoss/grimoirelab-graal): retrieval of data from source code
```
Comment (jgbarah) [inline README.md]: What about "source data analysis with external tools"?

Comment (valeriocos) [reply README.md]: sorry for the late reply, it seems that some notifications are forwared to my email with 1 day of delay

Comment (jgbarah) [reply README.md]: No problem. What do you think about my suggestion for that slight change in the text?

Comment (valeriocos) [reply README.md]: Yes, sure. I have just updated the PR.

**Pull request merged**